src/ol/render/vectorcontext.js

```diff
@@ -16,7 +16,6 @@ ol.render.VectorContext = function() {
 /**
  * Render a geometry.
  *
- * @abstract
```

Comment:
By removing `@abstract` from the methods, the compiler would no longer complain if you would create a child class which did not implement all of the methods. Is that what you want?